src/adagrams.js

```diff
@@ -1,7 +1,220 @@
+
+const letters = {
```

Comment:
Really really really smart structure for this data: It's the same letters, so you might as well combine the count and the score!